App/Views/EditFeedPage.xaml

```diff
@@ -35,39 +35,98 @@
     <ContentPage.Content>
         <StackLayout Padding="50,0">
 
+
+            <!-- Notifications -->
+            <Border StrokeShape="RoundRectangle 20"
+                    BackgroundColor="{DynamicResource LightDark}"
+                    WidthRequest="350">
+                <Grid ColumnDefinitions="50,auto,60"
+                      HorizontalOptions="Center">
+                    <!--Label-->
+                    <Label Text="&#xf0f3;"
+                           FontSize="25"
+                           TextColor="{StaticResource FontColor}"
+                            Grid.Column="0"
+                           VerticalOptions="Center"
+                           HorizontalOptions="Center"
+                           FontFamily="FaSolid"/>
+                    <Label Text="Notifications"
+                           FontSize="{StaticResource catFontSize}"
+                           TextColor="{StaticResource FontColor}"
+                            Grid.Column="1"
+                           VerticalOptions="Center"
+                           FontFamily="P-Bold"/>
+                    <Switch OnColor="{StaticResource PrimaryAccent}"
+                            ThumbColor="{StaticResource Primary}"
+                            HorizontalOptions="End"
+                            VerticalOptions="Center"
+                            IsToggled="{Binding FeedNotification}"
+                            WidthRequest="100"
+                            Grid.Column="2"
+                            Background="Transparent">
+                        <Switch.Triggers>
+                            <DataTrigger TargetType="Switch"
+                                         Binding="{Binding FeedNotification}"
+                                         Value="false">
+                                <Setter Property="ThumbColor" Value="{StaticResource FontColor}" />
+                            </DataTrigger>
+                        </Switch.Triggers>
+                    </Switch>
+                </Grid>
+
+            </Border>
+
             <!-- Edit name -->
-            <Grid RowDefinitions="auto,60">
-                <!--Label-->
-                <Label Text="Title"
+            <Border StrokeShape="RoundRectangle 15"
```

Comment:
All that seems unrelated but edit page update has everything to do with the implementation of the notification issue. As this one has to fit in the god awful design of this page